api/envoy/config/core/v3/substitution_format_string.proto

```diff
@@ -57,5 +60,29 @@ message SubstitutionFormatString {
     //  }
     //
     google.protobuf.Struct json_format = 2 [(validate.rules).message = {required: true}];
+
+    // A DataSource variant for text_format.
+    //
+    // The same semantics apply to the string/ that the referenced DataSource contains.
+    // This field takes precedent over text_format_source.
+    DataSource text_format_source = 5;
```

Comment:
Wait, this is `text_format_source`.  Which thing is it taking precedence over?

Reply:
this should say 'over text_format'